src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -26,6 +26,14 @@
       attach="#filing-history-list"
     />
 
+    <!-- Display for if a court order(s) have been filed -->
+    <v-card v-if="hasCourtOrders" class="my-6 pa-6" elevation="0">
+      <v-icon id="has-court-order-icon">mdi-gavel</v-icon>
```

Comment:
We haven't been setting an id on v-icons. If you need this for testing, please find a different way to select it.

Reply:
I was adding the id for this css because the color prop for v-icon wasn't using the custom $gray9 color.

Reply:
I still think you should target this v-icon differently... or, add the gray9 colour to the Vuetify scheme (in main.ts).

Reply:
Maybe `.v-icon.mdi-gavel` ?